src/components/SwitchContainer.ts

```diff
@@ -153,7 +156,9 @@ class SwitchContainer extends Component<SwitchContainerProps, SwitchContainerSta
     }
 
     private executeAction(actionname: string, guid: string) {
-        if (actionname && this.props.mxObject) {
+        const { mxObject, mxform } = this.props;
+
+        if (actionname && this.props.mxObject && actionname !== "onChangeNanoflow") {
```

Comment:
`mxObject` is already retrieved from `this.props` via destructuring, so Correct to `if (actionname && mxObject && actionname !== "onChangeNanoflow") `

Reply:
Done